assets/css/packages/s11-floating-links.css

```diff
@@ -0,0 +1,197 @@
+.s11-floating-links,
+.s11-floating-links::before,
+.s11-floating-links::after,
+.s11-floating-links *,
+.s11-floating-links *::before,
+.s11-floating-links *::after {
+	box-sizing: border-box;
+}
+
+.s11-floating-links {
+	position: fixed;
+	right: 48px;
```

Comment:
Can we override this position in BD? I think it's 48px in FF because of the screen options tab. But here, it would looks better if it aligned with the notifications. It matches for me with 20px for right and bottom.

Before:
<img width="406" alt="Screenshot 2023-08-04 at 6 41 23 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/9af83499-48ef-468c-8e6c-a9335ebe6dbe">

After:
<img width="358" alt="Screenshot 2023-08-04 at 6 41 11 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/7ed5e4b5-dae2-42fc-b98d-ee4165708450">

Reply:
This is almost there. It's just a bit too high now. Here are the current links with the right spacing:
<img width="192" alt="Screenshot 2023-08-29 at 5 53 05 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/4edd21a6-e918-4907-b149-af7d36186e90">

But this is spaced too high. It should be the same 16px as the above screenshot:
<img width="355" alt="Screenshot 2023-08-29 at 5 53 52 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/4a1821dc-684b-4e85-9767-b86f3e2f9be3">